libraries/lib-file-formats/AcidizerTags.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "sndfile.h"
+#include <optional>
+#include <string>
+#include <vector>
```

Comment:
These are not necessary to compile what is below.  If they are required in other places that include this header, I would move them there.